app/views/slot_types/index.html.erb

```diff
@@ -5,7 +5,7 @@
 <p class='index-slot-types-warning'><%= t('index_slot_types_warning') %></p>
 
 <div class='slot-type-apt-type-link-container'>
-  <% @agenda.place.appointment_types.each do |apt_type| %>
+  <% @agenda.place.appointment_types.select(&:with_slot_types?).each do |apt_type| %>
```

Comment:
Utiliser le scope

Reply:
aussi je te conseille de regarder les delegates pour éviter les chainages comme ça https://itnext.io/understanding-delegate-in-ruby-on-rails-i-wish-i-knew-before-5edd341bad47